test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -681,56 +680,50 @@ function source(::en_ρatke, ::BuoySource, atmos, args)
 end
 
 function source(::en_ρatke, ::DissSource, atmos, args)
-    @unpack env, l_mix, Diss₀ = args.precomputed.turbconv
+    @unpack env, Diss₀ = args.precomputed.turbconv
     gm = args.state
     en = args.state.turbconv.environment
     ρa₀ = gm.ρ * env.a
-    tke_en = enforce_positivity(en.ρatke) / gm.ρ / env.a
+    tke_en = enforce_positivity(en.ρatke) / ρa₀
     return -ρa₀ * Diss₀ * tke_en  # tke Dissipation
```

Comment:
in that case (i.e. below) this 
-ρa₀ * Diss₀ * tke_en
should be 
-en.ρatke* Diss₀ 
as well

Reply:
Yes, I will change this. It is good to have rectifying damping. Thanks!